MIGRATION.md

```diff
@@ -19,6 +19,23 @@ The deprecated `user.passport_id` data property has been removed.
 +tracking.click.init();
 ```
 
+`tracking.developer` has been removed, it has been replaced by the configuration parameter `test`. Set `test` to `true` to turn send events in test mode.
```

Comment:
>to turn send events in test mode.

what now? also what does test mode do? send the event but the event is ignored or bucketed differently by Spoor?

Reply:
```suggestion
The `tracking.developer` function has been removed, it has been replaced by a configuration parameter named `test`. Set the `test` configuration parameter to `true` to have similar behavior to the old `developer` mode.
```

Reply:
Nice!